runtime/src/lib.rs

```diff
@@ -226,7 +227,7 @@ parameter_types! {
         })
         .avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
         .build_or_panic();
-    pub const SS58Prefix: u16 = 2340;
+    pub const SS58Prefix: u16 = conf!(mainnet: 2340, testnet: 2340, devnet: 2300);
```

Comment:
Maybe _testnet_ env should use another chain id too?

Reply:
Many testers already use this 2340 chain ID for the testnet, it makes sense now to change the chain ID for mainnet (temporary) and discuss this config before starting mainnet / stagenet

Reply:
https://github.com/Atleta-network/atleta/issues/158